src/arrayview.rs

```diff
@@ -130,23 +128,69 @@ where
     }
 }
 
+impl<'a, T> From<&'a Vector<T>> for ArrayView<'a, T>
+where
+    T: for<'b> Struct<'b>,
+{
+    fn from(v: &'a Vector<T>) -> Self {
```

Comment:
I am not sure, but I think `'a` can be omitted in recent compilers. Could you please check?

Reply:
Doesn't seem to work since it becomes part of the type (basically there are two lifetimes here: the one that is part of the type and the one for the reference of the parameter... rustc does not seem to infer that both could be the same)